### PR TITLE
Scaffold docs for Intercom Web and Cloud Actions

### PR DESCRIPTION
**Pull request by kdaswani**
### Proposed changes
Intercom Web (Actions) and Intercom Cloud Mode (Actions) are in private beta. We have not invited any customers yet as Intercom is going to test themselves first. This is just the first version to scaffold the docs. ~I may make a couple more changes before next Tues (9/20) deploy but otherwise would be great to get this deployed next Tues.~ Additional change complete - this is ready to deploy as hidden with the auto-populated tables!

Since these two destinations are in Private Beta, can we make sure that the dossier and other tables (other versions of destination, settings, actions, fields) get pre-populated? Thank you!!

### Merge timing
-  ASAP

### CODE REVIEW

File: src/connections/destinations/catalog/actions-intercom-web/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+versions:
+  - name: 'Intercom Cloud Mode (Actions)'
+    link: '/docs/connections/destinations/catalog/actions-intercom-cloud'
+  - name: 'Intercom (Classic)'
+    link: '/docs/connections/destinations/catalog/intercom'
+---
+
+{% include content/plan-grid.md name="actions" %}
+
+Intercom is a customer communications platform. It shows you who is using your product and makes it easy to personally communicate with your users with targeted content, behavior-driven messages, and conversational support.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-intercom-web/index.md:16]:
```suggestion
Intercom is a customer communications platform that shows you who is using your product. Intercom also allows you to personally communicate with your users with targeted content, behavior-driven messages, and conversational support.
```

File: src/connections/destinations/catalog/actions-intercom-web/index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+When you use the Intercom Web (Actions) destination, Segment will load the [Intercom JavaScript library](https://developers.intercom.com/installing-intercom/docs/intercom-for-web){:target="_blank"} for you. The Intercom library enables you to track your user’s events on your website and interact with the Intercom messenger window.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-intercom-web/index.md:18]:
```suggestion
When you use the Intercom Web (Actions) destination, Segment loads the [Intercom JavaScript library](https://developers.intercom.com/installing-intercom/docs/intercom-for-web){:target="_blank"} for you. The Intercom library enables you to track your user’s events on your website and interact with the Intercom messenger window.
```

File: src/connections/destinations/catalog/actions-intercom-web/index.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+> info ""
+> The Intercom Web (Actions) destination is in beta and is in active development. Some functionality may change before it becomes generally available.
+
+## Benefits of Intercom Web Mode (Actions) vs Intercom Classic
+Intercom Web (Actions) provides the following benefits over the classic Intercom destination:
+
+- **Fewer settings.** Data mapping for actions-based destinations happens during configuration, which eliminates the need for most settings. 
+- **Clearer mapping of data.** Actions-based destinations enable you to define the mapping between the data Segment receives from your source, and the data Segment sends to the destination.
+- **Granular control over data sent.** You can customize the conditions under which the events will be sent to Intercom.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-intercom-web/index.md:28]:
```suggestion
- **Granular control over data sent.** You can customize the conditions under which the events are sent to Intercom. 
```

File: src/connections/destinations/catalog/actions-intercom-web/index.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+Intercom Web (Actions) provides the following benefits over the classic Intercom destination:
+
+- **Fewer settings.** Data mapping for actions-based destinations happens during configuration, which eliminates the need for most settings. 
+- **Clearer mapping of data.** Actions-based destinations enable you to define the mapping between the data Segment receives from your source, and the data Segment sends to the destination.
+- **Granular control over data sent.** You can customize the conditions under which the events will be sent to Intercom. 
+- **Selectively show the Intercom chat widget.**
+
+## Getting Started
+
+1. From the Segment web app, click **Catalog**.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-intercom-web/index.md:33]:
```suggestion
1. From the Segment web app, navigate to **Connections > Catalog**.
```

File: src/connections/destinations/catalog/actions-intercom-web/index.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+> info ""
+> The Intercom Web (Actions) destination is in beta and is in active development. Some functionality may change before it becomes generally available.
+
+## Benefits of Intercom Web Mode (Actions) vs Intercom Classic
+Intercom Web (Actions) provides the following benefits over the classic Intercom destination:
+
+- **Fewer settings.** Data mapping for actions-based destinations happens during configuration, which eliminates the need for most settings. 
+- **Clearer mapping of data.** Actions-based destinations enable you to define the mapping between the data Segment receives from your source, and the data Segment sends to the destination.
+- **Granular control over data sent.** You can customize the conditions under which the events will be sent to Intercom. 
+- **Selectively show the Intercom chat widget.**
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-intercom-web/index.md:29]:
```suggestion
- **Selectively shows the Intercom chat widget.**
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

**Comment by kdaswani** [comment]
@stayseesong thanks for your review! I added a couple of FAQs to the Intercom Cloud Mode docs based on some things we found in testing. This is ready to go for tomorrow's deploy please!

File: src/connections/destinations/catalog/actions-intercom-cloud/index.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+4. Select the source that will send data to Intercom Cloud Mode (Actions) and follow the steps to name your destination.
+5. On the **Settings** tab, authenticate with Intercom using OAuth.
+6. Follow the steps in the Destinations Actions documentation on [Customizing mappings](/docs/connections/destinations/actions/#customizing-mappings).
+7. Enable the destination and configured mappings.
+
+{% include components/actions-fields.html settings="true"%}
+
+{% include components/actions-fields.html%}
+
+## FAQ & Troubleshooting
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-intercom-cloud/index.md:47]:
```suggestion
## FAQs 
```

File: src/connections/destinations/catalog/actions-intercom-cloud/index.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+### Why is a company I created missing from my Intercom dashboard?
+If a company is created without an attached user, the company will not appear on Intercom's dashboard. This is expected. Once a user is attached to the company, it should appear in the list of companies.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-intercom-cloud/index.md:50]:
```suggestion
If a company is created without an attached user, the company won't appear on Intercom's dashboard. This is expected. Once a user is attached to the company, the company appears in the list of companies.
```

File: src/connections/destinations/catalog/actions-intercom-cloud/index.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+### Why isn’t a user getting attached to a company?
+When using the Identify Company action, Segment will first create or update a company. In the same action, Segment will also attach the user in your group call to that company. If the user does not exist in Intercom when the action runs, we will create/update the company but not attach the user. Please ensure the user is created in Intercom first.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-intercom-cloud/index.md:53]:
```suggestion
When using the Identify Company action, Segment first creates or updates a company. In the same action, Segment also attaches the user in your group call to that company. If the user does not exist in Intercom when the action runs, Segment will create and update the company but not attach the user. Please ensure the user is created in Intercom first.
```

**Review by stayseesong — APPROVED** [review]

**Pull request merged**